src/release-specification.test.ts

```diff
@@ -684,6 +684,700 @@ ${releaseSpecificationPath}
       });
     });
 
+    it('throws if there are any packages in the release with a major version bump using the word "major", but any of their dependents defined as "dependencies" are not listed in the release', async () => {
+      await withSandbox(async (sandbox) => {
+        const project = buildMockProject({
+          workspacePackages: {
+            a: buildMockPackage('a', {
+              hasChangesSinceLatestRelease: true,
+            }),
+            b: buildMockPackage('b', {
+              hasChangesSinceLatestRelease: false,
```

Comment:
See my note below about checking for changed dependents. In the context of this test, "b" doesn't have any changes, so I feel like this test shouldn't pass. What do you think about using `hasChangesSinceLatestRelease: true` in this and the other tests you've added? We may also want to add a new test to ensure that if "b" is NOT changed then the error doesn't get produced, but I can suggest that later in case you forget (there's a lot to keep track of here 😅 ).